Lib/test/libregrtest/cmdline.py

```diff
@@ -264,7 +264,9 @@ def _create_parser():
                        help='only write the name of test cases that will be run'
                             ' , don\'t execute them')
     group.add_argument('-P', '--pgo', dest='pgo', action='store_true',
-                       help='enable Profile Guided Optimization training')
+                       help='enable Profile Guided Optimization (PGO) training')
+    group.add_argument('--pgo-extended', action='store_true',
```

Comment:
I don't understand what "extended" means. I suggest to rename the option to "--pgo-all-tests" and rewrite the help to explain that it runs all tests by default.

Reply:
Think list.extend().  I.e. run more tests.  I didn't want to use --pgo-all because we don't actually run all the tests.  Some are excluded using the `support.PGO` flag.  However, I don't feel strongly so if you really like --pgo-all, I will change it.

Reply:
Hmm, --pgo-harder?  :smile: